lib/tbot/config/testdata/TestBotConfig_YAML/standard_config.golden

```diff
@@ -1,4 +1,9 @@
 version: v2
+onboarding:
+  token: my-gitlab-token
+  join_method: gitlab
+  gitlab:
+    env_var_name: MY_CUSTOM_ENV_VAR
```

Comment:
What do you think of making this `token_env_var_name` for clarity? Or is this clearly the only environment variable we could be referring to?

Reply:
[89a365c](https://github.com/gravitational/teleport/pull/54104/commits/89a365c04f8a6e5f8aa5492edeb7ed6b6a9548e0)